src/validation/user.js

```diff
@@ -0,0 +1,67 @@
+import joi from 'joi';
+
+export const CreateSchema = joi.object().keys({
```

Comment:
perhaps you should consider validating a name against a pattern for a full name too